docker/web/public/week4/employee.php

```diff
@@ -1,56 +1,87 @@
 <?php
 
+$charset = 'utf8';
+$dsn = sprintf('mysql:host=%s;dbname=%s;charset=%s', $_ENV['MYSQL_HOST'], $_ENV['MYSQL_DATABASE'], $charset);
+$options = [
+   PDO::ATTR_ERRMODE            => PDO::ERRMODE_EXCEPTION,
+   PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC,
+   PDO::ATTR_EMULATE_PREPARES   => false,
+];
+try {
+   /**
+    * $db pdo
+    */
+   $db = new PDO($dsn, $_ENV['MYSQL_USER'], $_ENV['MYSQL_PASSWORD'], $options);
+} catch (\PDOException $e) {
+   throw new \PDOException($e->getMessage(), (int)$e->getCode());
+}
+
+//$stmt = $db->query('SELECT *  FROM `employee` WHERE `id` = 1');
+//$results = $stmt->fetchAll();
+//print_r($results);
+
+// new code
+$id = 1;
+$stm = $db->prepare('SELECT *  FROM `employee` WHERE `id` = ?');
```

Comment:
FYI: when I have a query and instead of using a bunch of question marks, :employeeID
```  'SELECT * FROM `employee` WHERE `id` = :employeID
That way you can associate the variable with the key.